### PR TITLE
Support new ngrok

### PR DESCRIPTION
**Pull request by TalyaS**
Supports syntax for ngrok 2.x.

### CODE REVIEW

File: shoot.gemspec [hunk 1]
```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec", "~> 2"
 
   spec.add_dependency "selenium-webdriver", "~> 2"
-  spec.add_dependency "capybara", "~> 2"
+  spec.add_dependency "capybara", "2.4.4"
```
Comment (moriaam) [inline shoot.gemspec]: Why is the gem locked on this version?

File: lib/shoot/ngrok.rb [hunk 2]
```diff
@@ -7,8 +7,12 @@ class Ngrok
     extend Forwardable
     def_delegators :@process, :start, :stop, :exited?
 
-    def initialize(port = 3000)
-      @process = ChildProcess.build("ngrok", "-log=stdout", "-subdomain=#{subdomain}", port.to_s)
+    def initialize(options)
+      port = options[:port] || 3000
+      params = ["ngrok", "http", port.to_s, "-log=stdout", "-subdomain=#{subdomain}"]
```
Comment (moriaam) [inline lib/shoot/ngrok.rb]: Why is 'http' suddenly passed?

File: shoot.gemspec (hunk 1 above)
Comment (TalyaS) [inline shoot.gemspec]: to reflect that the gem does not support versions above it.
(not related to my change, actually)

File: lib/shoot/ngrok.rb (hunk 2 above)
Comment (TalyaS) [inline lib/shoot/ngrok.rb]: since the syntax for ngrok changed.
1.x: "ngrok 3000 <params>"
2.x: "ngrok http 3000 <params>"

**Comment by joaomilho** [comment]
There is an error on tests, did you take a look into it?

**Comment by joaomilho** [comment]
I'll add Travis here

File: lib/shoot/ngrok.rb (hunk 2 above)
Comment (tomerlevi444) [inline lib/shoot/ngrok.rb]: '-log=stdout' should be removed

**Pull request merged**